pkg/controller/backup_controller.go

```diff
@@ -281,6 +290,47 @@ func (c *backupController) getLocationAndValidate(itm *api.Backup, defaultBackup
 	return backupLocation, validationErrors
 }
 
+// getVolumeLocationAndValidate ensures:
+// - each VSL name in Spec VolumeSnapshotLocation exists as a VSL
+// - exactly 1 VSL per existing or default provider
+// - a given default provider's VSL name is added to the Spec VolumeSnapshotLocation if it does not exist as a VSL
+func (c *backupController) getVolumeLocationAndValidate(itm *api.Backup, defaultVSLs map[provider]string) []string {
```

Comment:
Personally speaking I find acronyms in the middle of variable names more difficult to read.  WDYT about just using `location` in place of `VSL` - since this function is all about snapshot locations, there's not really any ambiguity about what `location` means?